swap_meet/decor.py

```diff
@@ -1,2 +1,9 @@
-class Decor:
-    pass
+from .item import Item
+
+class Decor(Item):
+    def __init__(self, category="", condition=0):
+        self.category = "Decor"
+        self.condition = condition
```

Comment:
You can use inheritance here by invoking super()
```suggestion
     super().__init__(category="Decor",condition)
```